tests/unit_tests/test_session.py

```diff
@@ -62,7 +62,7 @@ def test_query_success(self, mock_token, mock_env):
                     conn=connector,
                     headers=headers,
                 ) as session:
-                    result = await session.fetch(me_query)
+                    result = await session.fetch(me_query, variables={})
```

Comment:
Should rather be `None`, no?

```suggestion
                    result = await session.fetch(me_query, variables=None)
```